meteor/sofie-server/model/entities/ad-lib-piece.ts

```diff
@@ -0,0 +1,50 @@
+import { TimelineObject } from './timeline-object'
+
+export interface AdLibPieceInterface {
+	id: string
+	rundownId: string
+	name: string
+	duration: number
+	timelineObjects: TimelineObject[]
+}
+
+export class AdLibPiece {
```

Comment:
Might have not understood the desired structure correctly, but shouldn't this class implement the `AdLibPieceInterface`?

Reply:
No, the reason for these interfaces is so that we can have a single parameter in the constructor instead x parameters.
Remember, more than three parameters, then you are most likely doing something wrong (not always, but most likely)

Reply:
We could also use a Builder, but that's a lot of boilerplate code compared to just writing an interface

Reply:
Hmm, sounds like the interface is an Dto.
If so, should the name of the interfaces not reflect this?